src/saucelabs/runner.js

```diff
@@ -95,7 +95,7 @@ export default class SauceLabsRunner extends EventEmitter {
     async _checkForFreeMachines () {
         var freeMachinesCount = await this._getFreeMachineCount();
 
-        if (freeMachinesCount)
+        if (freeMachinesCount > 0)
```

Comment:
It's not clear without the problem context why we can't use just `if (freeMachinesCount)`, please add the comment.